test/teiserver/protocols/tachyon/v1/tachyon_auth_test.exs

```diff
@@ -73,6 +73,8 @@ defmodule Teiserver.Protocols.V1.TachyonAuthTest do
     _tachyon_send(socket, %{"cmd" => "c.user.list_users_from_ids", "id_list" => friend_list, "include_clients" => true})
     [resp] = _tachyon_recv(socket)
     assert resp["cmd"] == "s.user.user_and_client_list"
+    [clients] = resp["clients"]
+    assert clients["sync"]
```

Comment:
Please assert what the value is, just to be safe

Reply:
Done